src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,63 @@
 package core.basesyntax;
 
+import java.util.LinkedList;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private LinkedList<Entry<K, V>>[] buckets;
+    private int size;
+
+    private static class Entry<K, V> {
+        private K key;
+        private V value;
+
+        Entry(K key, V value) {
+            this.key = key;
+            this.value = value;
+        }
+    }
+
```

Comment:
The `Entry` class is defined before the public methods of `MyHashMap`. According to the checklist, private methods and classes should be placed after public ones.